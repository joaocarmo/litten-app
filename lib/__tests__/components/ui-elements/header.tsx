import TestRenderer from 'react-test-renderer'
import ThemeProvider from '@components/theme/provider'
import { UIHeader } from '@ui-elements'

describe('Snapshot test for the "UIHeader" component', () => {
  it('renders correctly', () => {
    const element = TestRenderer.create(
      <ThemeProvider>
        <UIHeader>Some text here</UIHeader>
      </ThemeProvider>,
    ).toJSON()
    expect(element).toMatchSnapshot()
  })
})
