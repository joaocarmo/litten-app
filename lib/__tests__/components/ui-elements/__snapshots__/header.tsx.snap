// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test for the "UIHeader" component renders correctly 1`] = `
<Text
  style={
    Array [
      Object {
        "color": "#000000",
        "fontSize": 25.22845458984375,
        "fontWeight": "700",
        "marginBottom": 2,
        "marginTop": 2,
        "paddingBottom": 2,
        "paddingTop": 2,
      },
      undefined,
      undefined,
      undefined,
    ]
  }
>
  Some text here
</Text>
`;
